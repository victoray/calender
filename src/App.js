import React from 'react';
import NavBar from "./components/NavBar";
import {Container} from "semantic-ui-react";
import Calender from "./components/Calender";

function App() {
    return (
        <div className="App">
            <NavBar/>
            <Container>
                <Calender/>
            </Container>
        </div>
    );
}

export default App;
