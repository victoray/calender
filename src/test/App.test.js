import React from "react";
import {shallow, render} from "enzyme";
import App from "../App";

it("should render <App/>", () => {
    const component = shallow(<App/>);
    expect(component).toMatchSnapshot();
});