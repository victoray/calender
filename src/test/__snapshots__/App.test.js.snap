// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render <App/> 1`] = `
<div>
  <NavBar />
  <Container>
    <Calender />
  </Container>
</div>
`;
